articles/connections/apple-siwa/test-siwa-connection.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Test Sign In with Apple Configuration
+description: Learn how to test the Auth0 Sign In with Apple app configuration. 
+topics:
+  - authentication
+  - connections
+  - social
+  - apple
+contentType: how-to
+useCase:
+  - add-login
+  - customize-connections
+  - add-idp
+---
+
+# Test Sign In with Apple Configuration
+
+::: note
+If you are using the Classic Universal Login flow, or embedding `Lock.js` in your application, make sure you are using `Lock.js` version 11.16 or later.
+:::
```

Comment:
Might also want to add a note here about how native apps cannot be tested with browser-based flows?

Reply:
I actually added it to the Native article instead.